### PR TITLE
Added changelog, added NPM badge, and upgraded node+dependencies #35

### PR DESCRIPTION
**Pull request by patrick-th**
https://gitlab.com/jklabsinc/opslevel-bs/-/issues/5
https://gitlab.com/jklabsinc/opslevel-bs/-/issues/4

https://github.com/OpsLevel/backstage-plugin-backend/assets/58182580/9cdb9dd5-446a-45f2-8c5d-4fe2e2f244c4

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,6 +2,11 @@
 # yarn lockfile v1
```
Comment (katerberg) [inline yarn.lock:4]: These are a lot of changes to dependencies. I don't have a way to validate that this didn't break anything since we don't have a staging environment yet. Do you mind sharing a Loom to demo this please?

File: .node-version [hunk 2]
```diff
@@ -1 +1 @@
-16.18.0
+18.18.0
```
Comment (katerberg) [inline .node-version:1]: Bumping this would be cleaner to do in a different PR if possible (even though it's a good idea) since it's a bigger change and not needed for this ticket.

**Review by katerberg — CHANGES_REQUESTED** [review]
Ideally we can strip out the additional changes here or document a demo of it working.

The only hard blocker is to make the changelog follow the [keep a changelog format](https://keepachangelog.com/en/1.1.0/)

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+2023-10-04  Patrick  <patrick@opslevel.com>
```
[the remark this answers is not in the document]
Comment (patrick-th) [reply CHANGELOG.md:1]: Could you be more specific? I got that from that page -> Is there a standard changelog format? ->GNU changelog style guide

File: yarn.lock (hunk 1 above)
Comment (patrick-th) [reply yarn.lock:4]: Yeah I wasn't gonna merge this without testing locally first. I can record a video.

**Review by katerberg — APPROVED** [review]
Looks great! Thanks for updating the changelog. I defer to you on when it's tested enough to merge, but the code looks good.

The npm badge is a nice addition!
